x/relationships/client/cli/query.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/desmos-labs/desmos/x/relationships/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd adds the query commands
+func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+	profileQueryCmd := &cobra.Command{
```

Comment:
We should rename this `profileQueryCmd` with a more generic `cmd` I think, so that we don't have any problem moving it around if needed